projects/storefrontlib/src/lib/product/components/product-details/product-images/product-images.component.scss

```diff
@@ -1,96 +1,33 @@
-$cx-product-images-pdp-zoom-width: 26px !default;
-$cx-product-images-pdp-zoom-height: 26px !default;
-$cx-product-images-pdp-zoom-trbl-margin: 10px 10px 0 0 !default;
-$cx-product-images-pdp-small-border: 2px solid !default;
-$cx-product-images-pdp-small-border-color: 'light' !default;
-$cx-product-images-pdp-small-border-color-hover: 'primary' !default;
-$cx-product-images-pdp-small-trbl-margin: 0 25px 0 0 !default;
-$cx-product-images-pdp-small-trbl-padding: 10px 10px 10px 10px !default;
-$cx-product-images-pdp-item-image-width: 100% !default;
-$cx-product-images-pdp-small-height: 100px !default;
-$cx-product-images-pdp-small-width: 100px !default;
-$cx-product-images-pdp-small-img-height: 100px !default;
-$cx-product-images-pdp-small-img-width: 100px !default;
-$cx-product-images-pdp-small-img-border-color: 'light' !default;
-$cx-product-images-pdp-items-small-lastchild-trbl-margin: 0 0 0 0 !default;
-$cx-product-images-pdp-mobile-small-trbl-margin: 0 15px 0 0 !default;
-$cx-product-images-pdp-mobile-small-height: 80px !default;
-$cx-product-images-pdp-mobile-small-width: 80px !default;
-.cx-product-images {
-  &__item-image {
-    width: $cx-product-images-pdp-item-image-width;
-  }
-  &__items {
-    text-align: center;
-    margin: $cx-product-images-pdp-items-trbl-margin;
-  }
-  &__item-zoom--icon {
-    @include var-color('color', $cx-product-images-pdp-zoom-color);
-    border: $cx-product-images-pdp-zoom-border;
-    border-radius: $cx-product-images-pdp-zoom-border-radius;
-    display: block;
-    width: $cx-product-images-pdp-zoom-width;
-    height: $cx-product-images-pdp-zoom-height;
-    float: right;
-    margin: $cx-product-images-pdp-zoom-trbl-margin;
-    cursor: pointer;
-    &:after {
-      content: '+';
-      bottom: 13px;
-      position: relative;
-      left: 3px;
-      font-size: 29px;
-    }
-    &:hover {
-      @include var-color('color', $cx-product-images-pdp-zoom-color-hover);
-    }
-  }
-  &__item--small {
-    margin: $cx-product-images-pdp-small-trbl-margin;
-    border: $cx-product-images-pdp-small-border;
-    @include var-color(
-      'border-color',
-      $cx-product-images-pdp-small-border-color
-    );
-    width: $cx-product-images-pdp-small-width;
-    height: $cx-product-images-pdp-small-height;
-    display: inline-block;
-    &:hover {
-      @include var-color(
-        'border-color',
-        $cx-product-images-pdp-small-border-color-hover
-      );
-    }
-    &:last-child {
-      margin: $cx-product-images-pdp-items-small-lastchild-trbl-margin;
-    }
-    @include media-breakpoint-down(sm) {
-      width: $cx-product-images-pdp-mobile-small-width;
-      height: $cx-product-images-pdp-mobile-small-height;
-      margin: $cx-product-images-pdp-mobile-small-trbl-margin;
+:host {
+  display: flex;
+  flex-direction: var(--cx-flex-direction, column);
+
+  --cx-border-color: var(--cx-light);
+  --cx-border-width: 0.1vw;
+}
+
+.thumbs {
+  display: flex;
+  justify-content: flex-start;
```

Comment:
Should these be variables too. Maybe not the display. Also in order to achieve your "Amazon" look you would need to have a variable for the `flex-direction`.

Reply:
Not needed actually, you can apply the following rules on app level:

```
cx-product-images {
  --cx-flex-direction: row-reverse;
  justify-content: space-between;

  .thumbs {
    flex-direction: column;
  }
}
```

Reply:
True, actually since `flex-direction` isn't defined you can just add it in the styles.